.npmignore

```diff
@@ -0,0 +1,86 @@
+# Logs
+/build
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (https://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
+node_modules/
+jspm_packages/
+
+# TypeScript v1 declaration files
+typings/
+
+# Optional npm cache directory
+.npm
+
+# Optional eslint cache
+.eslintcache
+
+# Optional REPL history
+.node_repl_history
+
+# Output of 'npm pack'
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
+# dotenv environment variables file
+.env
+
+# next.js build output
+.next
+
+/lib
+
+/dist
+
+/cjs
```

Comment:
cjs, esm, iife should not be ignored by npm, they are build files for this project.

it is okay to ignore tests, scripts and db, because they are not utilized by clients

Reply:
So I am going to remove those from .npmignore. Anyway, cjs and esm are in .gitignore, so they are being ignored now without using npmignore ([How to ignore files from your npm package](https://zellwk.com/blog/ignoring-files-from-npm-package/).

Reply:
nice catch @mastepanoski , we should utilize package.json files to specify what is included in the package.

Reply:
can you install the latest version of the build tool, `@teclone/rollup-all`. it outputs the build to a `/build` directory.

then include this build directory to package.json files.

Reply:
The build directory is not generated anymore. Try to remove it, then do yarn build, and you got the cjs, esm, iife and umd directory in the project root. Try to add rimraf to the script: "build": "rimraf ./build && yarn generateLocales && rollup-all" to clean the old directory, and do yarn build again. So using npmignore to blacklist is a better approach until you can assure that the older directories are cleaned and the new ones are generated in the desired way.

Reply:
@mastepanoski the latest version of the package does now. all you need is to install the latest version